packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -781,27 +786,67 @@ export const ListBase = CollectionWidget.inherit({
     }
   },
 
-  _renderGroup(index, group) {
-    const $groupElement = $('<div>')
-      .addClass(LIST_GROUP_CLASS)
-      .appendTo(this._getItemsContainer());
+  _setGroupAria($group, groupHeaderId): void {
+    const { collapsibleGroups } = this.option();
 
-    const id = `dx-${new Guid().toString()}`;
     const groupAria = {
-      role: 'group',
+      role: collapsibleGroups ? undefined : 'group',
       // eslint-disable-next-line spellcheck/spell-checker
-      labelledby: id,
+      labelledby: collapsibleGroups ? undefined : groupHeaderId,
     };
 
-    this.setAria(groupAria, $groupElement);
+    this.setAria(groupAria, $group);
+  },
+
+  _updateGroupHeaderAriaExpanded($groupHeader, expanded): void {
+    this.setAria({ expanded }, $groupHeader);
+  },
+
+  _setGroupHeaderAria($groupHeader, listGroupBodyId): void {
+    const { collapsibleGroups } = this.option();
+
+    const groupHeaderAria = {
+      role: collapsibleGroups ? 'button' : undefined,
+      expanded: collapsibleGroups ? true : undefined,
+      controls: collapsibleGroups ? listGroupBodyId : undefined,
```

Comment:
[minor]
now "collapsibleGroups" is copied and pasted several times, i'd prefer clear `if` using

Reply:
Right now yes, but I believe each of these attributes should be configurable separately. If you use ‘if’, you will configure all attributes at once.